Gemfile

```diff
@@ -67,6 +66,13 @@ group :development do
   #gem 'rack-mini-profiler'
 end
 
+group :test do
+  gem 'minitest-reporters'
+  gem 'shoulda', '~> 3.5'
+  gem 'shoulda-matchers', '~> 2.0'
+  gem 'rails-controller-testing'
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem rails-controller-testing should appear before shoulda-matchers.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.